### ISSUE TITLE
Wrong message displayed when damaging a "mimicked" object

### ISSUE THREAD

**Issue by angband-import** [angband/angband#3814]
**Reported by draconisPW on 30 May 2013 12:49 UTC**
Using a wand of cold balls on a potion will give the following message: "The potion shatters! The potion mimic freezes and shatters!"

Easy fix: use become_aware() in project_o().

```
	/* Artifacts, and other objects, get to resist */
	if (is_art || ignore)
	{
		/* Observe the resist */
		if (o_ptr->marked && !squelch_item_ok(o_ptr))
		{
			msg("The %s %s unaffected!",
				o_name, (plural ? "are" : "is"));
		}
	}

	/* Reveal mimics */
	else if (o_ptr->mimicking_m_idx)
                become_aware(cave_monster(cave, o_ptr->mimicking_m_idx));
	

	/* Kill it */
	else
	{
		/* Describe if needed */
		if (o_ptr->marked && note_kill && !squelch_item_ok(o_ptr))
		{
			msgt(MSG_DESTROY, "The %s%s", o_name, note_kill);
		}

		/* Delete the object */
		delete_object_idx(this_o_idx);

		/* Redraw */
		cave_light_spot(cave, y, x);
	}
```

**Comment by angband-import** [comment]
**Modified by takkaria on 30 Aug 2013 03:36 UTC**

**Comment by angband-import** [comment]
**Comment by takkaria on 16 Oct 2013 09:18 UTC**
Fixed in b020e2c, pull request https://github.com/angband/angband/pull/276
